test/fixtures/IL-2022.json

```diff
@@ -0,0 +1,290 @@
+  },
+  {
+    "date": "2022-04-23 00:00:00 -0600",
+    "start": "2022-04-22T15:00:00.000Z",
+    "end": "2022-04-23T15:00:00.000Z",
+    "name": "Mimouna",
+    "type": "observance",
+    "note": "Unpaid leave upon request",
+    "rule": "22 Nisan",
+    "_weekday": "Sat"
+  },
+  {
+    "date": "2022-04-28 00:00:00 -0600",
+    "start": "2022-04-27T15:00:00.000Z",
+    "end": "2022-04-28T15:00:00.000Z",
+    "name": "Holocaust Remembrance Day",
+    "type": "observance",
+    "note": "National remembrance day, business as usual except places of public entertainment",
+    "rule": "27 Nisan",
+    "_weekday": "Thu"
+  },
+  {
+    "date": "2022-05-05 00:00:00 -0600",
+    "start": "2022-05-04T15:00:00.000Z",
+    "end": "2022-05-05T15:00:00.000Z",
+    "name": "Fallen Soldiers and Victims of Terrorism Remembrance Day",
+    "type": "observance",
+    "note": "National remembrance day, business as usual except places of public entertainment",
+    "rule": "4 Iyyar",
+    "_weekday": "Thu"
+  },
+  {
+    "date": "2022-05-06 00:00:00 -0600",
+    "start": "2022-05-05T15:00:00.000Z",
+    "end": "2022-05-06T15:00:00.000Z",
+    "name": "Independence Day",
+    "type": "public",
+    "rule": "5 Iyyar",
+    "_weekday": "Fri"
+  },
+  {
+    "date": "2022-05-09 00:00:00",
+    "start": "2022-05-08T21:00:00.000Z",
+    "end": "2022-05-09T21:00:00.000Z",
+    "name": "Victory in Europe Day",
+    "type": "observance",
+    "rule": "05-09",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2022-05-11 00:00:00 -0600",
+    "start": "2022-05-10T15:00:00.000Z",
+    "end": "2022-05-11T15:00:00.000Z",
+    "name": "Herzl Day",
+    "type": "public",
+    "rule": "10 Iyyar",
+    "_weekday": "Wed"
+  },
+  {
+    "date": "2022-05-19 00:00:00 -0600",
+    "start": "2022-05-18T15:00:00.000Z",
+    "end": "2022-05-19T15:00:00.000Z",
+    "name": "Lag BaOmer",
+    "type": "school",
+    "rule": "18 Iyyar",
+    "_weekday": "Thu"
+  },
+  {
+    "date": "2022-05-29 00:00:00 -0600",
+    "start": "2022-05-28T15:00:00.000Z",
+    "end": "2022-05-29T15:00:00.000Z",
+    "name": "Jerusalem Day",
+    "type": "observance",
+    "note": "Optional paid leave",
+    "rule": "28 Iyyar",
+    "_weekday": "Sun"
+  },
+  {
+    "date": "2022-06-05 00:00:00 -0600",
+    "start": "2022-06-04T15:00:00.000Z",
+    "end": "2022-06-05T15:00:00.000Z",
+    "name": "Shavuot",
+    "type": "public",
+    "rule": "6 Sivan",
+    "_weekday": "Sun"
+  },
+  {
+    "date": "2022-07-16 00:00:00 -0600",
+    "start": "2022-07-15T15:00:00.000Z",
+    "end": "2022-07-16T15:00:00.000Z",
+    "name": "Seventeenth of Tamuz, fast",
+    "type": "observance",
+    "rule": "17 Tamuz",
+    "_weekday": "Sat"
+  },
+  {
+    "date": "2022-07-28 00:00:00 -0600",
+    "start": "2022-07-27T15:00:00.000Z",
+    "end": "2022-07-28T15:00:00.000Z",
+    "name": "Jabotinsky Day",
+    "type": "public",
+    "rule": "29 Tamuz",
+    "_weekday": "Thu"
+  },
+  {
+    "date": "2022-08-06 00:00:00 -0600",
```

Comment:
Tisha B'Av 2020 is on July 7

https://www.timeanddate.com/holidays/israel/2022

Reply:
Should be solved